app.html

```diff
@@ -277,7 +277,7 @@
 var style_lines;
 var named_styles = {};
 var tracks_listed = false;
-
+var has_dimblade = false;
```

Comment:
What fork(s) of ProffieOS has "get_dim" and "set_dim"?

Reply:
https://github.com/artandtechvof/ProffieOS/blob/dimblade/blades/dim_blade.h

Reply:
That's just the dimblade() helper, where are the commands?
Also, is this something you're going to try to merge into the main ProffieOS?

Reply:
I would like to add it into the main code, yes. Will make a pull request for that.
The DimBladeWrapper now also uses Saberbase
SaberBase has the SaberBase::SetBrightness and SaberBase::GetCurrentBrightness and DimChangeMode (like the colorchange mode)
prop_base has the actual brightness adjustment functions and the added serial commands for get_dim and (set) dim.
the

https://github.com/artandtechvof/ProffieOS/tree/dimblade